spring-cloud-dataflow-docs/src/main/asciidoc/architecture.adoc

```diff
@@ -35,7 +35,8 @@ There is a deployer Service Provider Interface (SPI) that enables you to extend
 
 The component that is responsible for deploying applications to a runtime is the Data Flow Server.  There is a Data Flow Server executable jar provided for each of the target runtimes.  The Data Flow server is responsible for interpreting
 
-* A stream DSL that describes the logical flow of data through multiple applications.
+* A stream DSL that describes the logical flow of data through multiple long lived applications.
+* A composed task DSL that describes the logical flow of data through multiple short lived applications.
```

Comment:
It isn't really a DSL, more like the running of a single app....  But it is worth mentioning this, doesn't have to mention the DSL.